### PR TITLE
Master milestone7 review

### PR DESCRIPTION
**Pull request by FR4NK-W**
review PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,266 @@
...
+  Solution: always use the latest path returned by send, e.g. `path = send(buffer, path)`.
+
+- **Using expired path (server).** When using `send(buffer, path)` with an expired `ResponsePath`, the channel will
+  simple send it anyway.
+
+## Configuration
+
+### Bootstrapping / daemon
+In order to find paths and connect to the local AS, the application needs either a (local) 
+installation of the Scion Daemon (see here) or some other means to
```
Comment (marcfrei) [inline README.md:212]: SCION

File: README.md [hunk 2]
```diff
@@ -1,3 +1,266 @@
...
+get bootstrap information.
```
Comment (marcfrei) [inline README.md:213]: Imprecise/overloaded wording: The bootstrap info consists of the topology.json and the certs. Usually you thus need the bootstrap info plus the daemon to get running. The daemon can't do the bootstrapping on its own.

**Review by marcfrei — COMMENTED** [review]
🚀

File: README.md [hunk 3]
```diff
@@ -1,3 +1,266 @@
...
-Java API for [SCION](https://scion.org)
 
+
+***Not officially released yet***
+
+# SCION Java client
+
+This library is pure Java network stack for using [SCION](https://scion.org). More information about SCION can be found [here](https://docs.scion.org).
+It provides functionality similar to 
+[snet (golang)](https://pkg.go.dev/github.com/scionproto/scion/pkg/snet),
```
[the remark this answers is not in the document]
Comment (marcfrei) [reply README.md:9]: On next line too.

File: doc/Design.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+### Comments
+* Selectors: Implementing a selectable Channel intentionally requires reimplementing
+  the whole Selector infrastructure, see also https://bugs.openjdk.org/browse/JDK-8191884.
+
+## DatagramSocket
+
+**DatagramSockets are currently not supported and may never be supported**. 
+DatagramSockets have no means of handling paths transparently (see discussion above).
+That means we would need additional functions for sending/receiving Scion packets.
```
Comment (marcfrei) [inline doc/Design.md:110]: SCION

File: src/test/resources/topologies/ETH.json
Comment (marcfrei) [inline src/test/resources/topologies/ETH.json:1]: Remove this file

**Review by marcfrei — CHANGES_REQUESTED** [review]

File: src/test/resources/topologies/ETH.json
Comment (tzaeschke) [reply src/test/resources/topologies/ETH.json:1]: Done!

**Comment by tzaeschke** [comment]
@marcfrei Thanks a lot!

**Review by marcfrei — APPROVED** [review]

**Pull request closed without merging**